book/data_viz/plotting_data.nim

```diff
@@ -7,18 +7,266 @@ nbUseNimibook
 nbText: """
 # Plotting data using [ggplotnim](https://github.com/Vindaar/ggplotnim)
 
-We'll plot some pretty plots!
+In this tutorial we will introduce `ggplotnim`, a Nim plotting library heavily inspired
+by the great R library [ggplot2](https://ggplot2.tidyverse.org).
 
-## Get df
+This will be kept rather brief, but we will discuss the philosophy of the syntax, look
+at a reasonably complex plotting example that we deconstruct and finish of by (hopefully)
+coming to the conclusion that the ggplot-like syntax is rather elegant.
 
-Get some DF in here.
+For this tutorial you should have read the data wrangling introduction to `Datamancer` or
```

Comment:
Would be nice with a link to these tutorials in the future when we have settled down on the structure of the repo :)

Reply:
Agreed!